visualization/dashboard/src/MLIDashboard/Controls/DatasetExplorerTab/DatasetExplorerTab.tsx

```diff
@@ -108,6 +109,7 @@ export class DatasetExplorerTab extends React.PureComponent<IDatasetExplorerTabP
         this.setSelectedCohort = this.setSelectedCohort.bind(this);
         this.toggleCalloutOpen = this.toggleCalloutOpen.bind(this);
         this.closeCallout = this.closeCallout.bind(this);
+        this.editCohort = this.editCohort.bind(this);
```

Comment:
doesn't this need to take in an index on the bind() call?  or why is this called?

Reply:
Welcome to react and 'this' context in javascript! Set all the functions of a class to be defined as bound to the instance in the constructor so that when a function is called in a callback, it has the correct 'this' instance available...